index.js

```diff
@@ -1,94 +1,62 @@
 var vm = require('vm');
 var requestModule = require('request');
-var jar = requestModule.jar();
-
-var request      = requestModule.defaults({jar: jar}); // Cookies should be enabled
-var UserAgent    = 'Ubuntu Chromium/34.0.1847.116 Chrome/34.0.1847.116 Safari/537.36';
-var Timeout      = 6000; // Cloudflare requires a delay of 5 seconds, so wait for at least 6.
-var cloudscraper = {};
-var MaxChallengesToSolve = 3; // Support only this max challenges in row. If CF returns more, throw an error
-
-/**
- * Performs get request to url with headers.
- * @param  {String}    url
- * @param  {Function}  callback    function(error, response, body) {}
- * @param  {Object}    headers     Hash with headers, e.g. {'Referer': 'http://google.com', 'User-Agent': '...'}
- */
-cloudscraper.get = function(url, callback, headers) {
-  performRequest({
-    method: 'GET',
-    url: url,
-    headers: headers
-  }, callback);
-};
-
-/**
- * Performs post request to url with headers.
- * @param  {String}        url
- * @param  {String|Object} body        Will be passed as form data
- * @param  {Function}      callback    function(error, response, body) {}
- * @param  {Object}        headers     Hash with headers, e.g. {'Referer': 'http://google.com', 'User-Agent': '...'}
- */
-cloudscraper.post = function(url, body, callback, headers) {
```

Comment:
Request has a formData option so I wasn't sure why this was being used. Is there some reason to prefer this over request's options?

Reply:
No special reason behind that, I didn't look thoroughly into request's api